hack/verify-all.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
I feel this file is not needed as of now, as we have only one verify to run. Just update the Makefile to add
```
.PHONY: verify-boilerplate
verify-boilerplate: ## Verify boilerplate text exists in each file
	 ./hack/verify-boilerplate.sh
```
And validate that it works by running
`make verify-boilerplate`